go.mod

```diff
@@ -3,4 +3,5 @@ module github.com/anz-bank/go-course
 require (
 	github.com/stretchr/testify v1.3.0
 	github.com/syndtr/goleveldb v1.0.0
+
```

Comment:
revert this change please.

Reply:
Done, sorry, not sure how the empty line got added.